scripts/check_firestore_core_api_absl.py

```diff
@@ -34,16 +34,16 @@ def diff_with_absl(revision, patterns):
   """Finds diffs containing 'absl' since a revision from specified path
   pattern.
   """
-
   # git command to print all diffs that has 'absl' in it.
-  command = ['git', 'diff', '-z', '-G', 'absl', revision, '--']
+  command = ['git', 'diff', '-G', 'absl', revision, '--']
   command.extend(patterns)
-  command_trace.log(command)
+  _logger.debug(command)
   return six.ensure_text(subprocess.check_output(command))
 
 
 def main():
-  parser = argparse.ArgumentParser(description='Lint source files.')
+  parser = argparse.ArgumentParser(
+      description='Check Absl usaging in firestore/core/api')
```

Comment:
Nits:
- `s/usaging/usage/`;
- please use the `patterns` variable instead of hardcoding `api/` here.

Reply:
Done.